stanford/bin/dump-logs

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+rsync_sudo() {
+    rsync --rsync-path='sudo rsync' -avh --progress ${@}
+}
+rsync_logs() {
+    machine=${1}
+    destination=${2}
+    shift; shift;
+    rsync_sudo ${@} "${machine}:/var/log/" "${destination}"
+    rsync_sudo ${@} "${machine}:/edx/var/log/" "${destination}/edx"
+}
+destination_root=/edx/var/log
+deployments="cme"
+clusters="analytics-api build certs coursegraph deploy director edxapp esforum forum insights jump maintenance nat notifier rabbitmq scheduler worker xqueue"
+for deployment in ${deployments}; do
+    for cluster in ${clusters}; do
+        for number in {0..4}; do
+            box="${cluster}${number}"
+            machine=${box}.${deployment}.vpc
+            if ssh -q -o ConnectTimeout=1 "${machine}" 'exit' 2>/dev/null; then
+                echo "Dumping logs from: ${machine}"
+                destination_box=${destination_root}/${deployment}/${box}
+                mkdir -p "${destination_box}"
```

Comment:
Ran into a permission issue on this command because creating directories at `/edx/var/log` requires sudo. After I added that the rsync commands complained about needing sudo permissions to match. Alternatively we could just create the log directory as your user account rather than root?

Reply:
@kluo Hrmm... Yeah, maybe try just creating that directory in advance as your normal user and then running without `sudo`?

Reply:
This should be resolved now, given the custom output directory functionality added above.

Reply:
Cool, worked for me